### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by SashaGT2** [Prodigy-Hacking/ProdigyMathGameHacking#497] [(B) Bug Discussion]
I've  just downloaded the official hack and I've auto received member and im kind of freaking out. can i have conformation that im not being charged. Thanks

p.s: is it possible to deactivate the member hack
p.p.s: how do i delete the instant kill thing

**Comment by PatheticMustan** [comment]
1. No
2. Disable the plugin lol
3. Refresh the page

**Reply by SashaGT2** [reply]
thanks
